tests/posts_tests/test_search_views.py

```diff
@@ -0,0 +1,50 @@
+import pytest
+from pytest_django.asserts import assertTemplateUsed
+
+
+@pytest.mark.django_db
+class TestViews:
+
+    search_url = '/search/?searched='
+
+    def test_searching_usernames_posts_view(self, client, persist_user, persist_post):
+        '''test if searching for a username does find his post,
```

Comment:
Maybe something like

```suggestion
        '''test searching for an exist post by username,
```

Reply:
fixed